ui/apps/platform/cypress/integration/integrations/notifiers.test.js

```diff
@@ -59,7 +59,18 @@ describe('Notifier Integrations', () => {
             cy.get(selectors.buttons.test).should('be.disabled');
             cy.get(selectors.buttons.save).should('be.disabled');
 
-            // Step 2, check fields for invalid formats
+            // Step 2, check conditional fields
+
+            // Step 2.1, enable container IAM role, this should remove the AWS credentials fields
+            getInputByLabel('Use container IAM role').click();
+            cy.get('[id=notifier.awsSecurityHub.credentials.accessKeyId]').should('not.exist');
```

Comment:
Here is an alternative that asserts visible text instead of invisible `id` attribute:
```js
            cy.get(
                `.pf-c-form__group:has('.pf-c-form__control:contains("Service Account Key (JSON)")') textarea`
            ).should('not.exist');
```

Reply:
For you info, error was lack of quote marks: `'[id=notifier.awsSecurityHub.credentials.accessKeyId]'`

Reply:
As follow up, UI team needs to rewrite `getInputByLabel` helper function to work for both:
* action or positive assertion
* negative assertion (which does not work now)

Because the current function does more than `input` elements (like `textarea` element) below, we might need to generalize it, or split it, or both.